docs/javascript/v2/how--to-guides/record-and-live-stream/hls/hls-timed-metadata.mdx

```diff
@@ -17,7 +17,7 @@ Currently there is a limit to send at max 3 metadata at time.
 
 ```javascript
 export interface HLSTimedMetadata {
-	payload: string; // limited to 100 chars, no " (double quotes) allowed
+	payload: string; // limited to 100 chars
```

Comment:
this needs to be updated in the limitations section also. 

can you also verify that all of the limitations are valid?

Reply:
We only have two validations on payload:
- Max 100 chars.
- no double quotes allowed
cc: @rtpriya @amar-1995

Reply:
but we can remove the double quotes validation - https://100ms-live.slack.com/archives/C0409JW6Y75/p1677049901459419

Reply:
@saikatmitra91 Currently there is a double quotes validation check at the backend. So if I make call payload as `'{"a":1}'` it will throw error